smriprep/workflows/anatomical.py

```diff
@@ -620,3 +631,55 @@ def _pop(inlist):
     if isinstance(inlist, (list, tuple)):
         return inlist[0]
     return inlist
+
+
+def _aseg_two_three():
+    import numpy as np
+    # Base struct
+    aseg_lut = np.zeros((256,), dtype="int")
+    # GM
+    aseg_lut[3] = 1
+    aseg_lut[8:14] = 1
+    aseg_lut[17:21] = 1
+    aseg_lut[26:40] = 1
+    aseg_lut[42] = 1
+    aseg_lut[47:73] = 1
+
+    # CSF
+    aseg_lut[4:6] = 3
+    aseg_lut[14:16] = 3
+    aseg_lut[24] = 3
+    aseg_lut[43:45] = 3
+    aseg_lut[72] = 3
+
+    # WM
+    aseg_lut[2] = 2
+    aseg_lut[7] = 2
+    aseg_lut[16] = 2
+    aseg_lut[28] = 2
+    aseg_lut[41] = 2
+    aseg_lut[46] = 2
+    aseg_lut[60] = 2
+    aseg_lut[77:80] = 2
+    aseg_lut[250:256] = 2
+    return tuple(aseg_lut)
+
+
+def _split_segments(in_file):
+    from pathlib import Path
+    import numpy as np
+    import nibabel as nb
+
+    segimg = nb.load(in_file)
+    data = np.int16(segimg.dataobj)
+    hdr = segimg.header.copy()
+    hdr.set_data_dtype('uint8')
+
+    out_files = []
+    for i, label in enumerate(("CSF", "WM", "GM")):
```

Comment:
Clearer than adding 1 later, IMO:

```suggestion
    for i, label in enumerate(("CSF", "WM", "GM"), 1):
```